R/as_polars.R

```diff
@@ -36,9 +39,28 @@ as_polars_df.default = function(x, ...) {
 
 
 #' @rdname as_polars_df
+#' @param rownames How to treat existing row names of a data frame:
+#'  - `NULL`: Remove row names. This is the default.
+#'  - A string: The name of a new column, which will contain the row names.
+#'    If `x` already has a column with that name, the existing column will be removed.
```

Comment:
Shouldn't it error instead? I think it's more likely people give the same name by accident rather than knowingly replacing a column. They can just drop the column before converting to DataFrame

Reply:
I have no strong opinion.
Let's switch to raise error.